README.md

```diff
@@ -10,8 +10,7 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10797541.svg)](https://zenodo.org/doi/10.5281/zenodo.10797541)
 
-**TrixiParticles.jl** is a numerical simulation framework designed for particle-based numerical methods, with an emphasis on multiphysics applications, written in [Julia](https://julialang.org).
-A primary goal of the framework is to be user-friendly for engineering, science, and educational purposes. In addition to its extensible design and optimized implementation, we prioritize the user experience, including installation, pre- and postprocessing.
+**TrixiParticles.jl** is a high-performance particle simulation solver framework written in [Julia](https://julialang.org). It offers intuitive configuration, robust pre- and post-processing, and seamless scalability from CPU to Exascale-level GPU computing, using KernelAbstractions.jl for GPU manufacturer-agnostic support and MPI (WIP) for parallel processing. The extensible architecture allows users to incorporate additional particle methods easily, making TrixiParticles.jl ideal for research and educational purposes.
```

Comment:
1. "particle simulation solver framework" sounds weird IMO, and it sounds like we are simulating particles, which is misleading.
2. We don't have MPI on main and "seamless scalability from CPU to Exascale-level GPU computing" is a vast overstatement.
3. It does not answer the question in #630 about the "target audience". I tried to add this below:
```suggestion
**TrixiParticles.jl** is a high-performance numerical simulation framework using particle-based numerical methods to simulate complex multiphysics problems and it is written in [Julia](https://julialang.org).
TrixiParticles.jl is targeted towards two groups of users:
- Users researching particle-based methods who want to develop new techniques or modify existing schemes.
- Users who want to simulate complex multiphysics and do not necessarily interested in the underlying numerical methods.

It offers intuitive configuration, robust pre- and post-processing, and vendor-agnostic GPU-support based on the Julia package [KernelAbstractions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl). The extensible architecture allows users to incorporate additional particle methods easily, making TrixiParticles.jl ideal for research and educational purposes.
```